src/EventStore.Client/Exceptions/UserNotFoundException.cs

```diff
@@ -2,9 +2,20 @@
 
 #nullable enable
 namespace EventStore.Client {
+	/// <summary>
+	/// The exception that is thrown when an operation is performed on an internal user that does not exist.
```

Comment:
```suggestion
	/// The exception that is thrown when an operation is performed by an internal user that does not exist.
```

Reply:
IMO the original wording is correct, this is thrown e.g. when I change the password of a non existing user.